dataactbroker/fsrs.py

```diff
@@ -146,6 +147,16 @@ def new_client(service_type):
             password=config['password'],
             timeout=300)
 
+    proxy_options = {}
+    http_proxy = os.environ.get('HTTP_PROXY')
+    if http_proxy:
+        proxy_options['http'] = http_proxy
+    https_proxy = os.environ.get('HTTPS_PROXY')
+    if https_proxy:
+        proxy_options['https'] = https_proxy
+    if proxy_options:
+        options['proxy'] = proxy_options
```

Comment:
`no_proxy` should be set the same way, so that you don't accidentally try and use a proxy for something that is no_proxy'd

Reply:
Good catch. I don't know if the suds package accounts for no_proxy (at least it isn't well documented). I'm gonna try another solution first, one that bypasses suds' own proxy settings.

Reply:
There we go. I had to somewhat tweak this solution but seems like it's working.
https://stackoverflow.com/a/64974205/4166584